### ISSUE TITLE
Why making an open letter if there is no discussion?

### ISSUE THREAD

**Issue by waltercool** [rms-open-letter/rms-open-letter.github.io#2259]
While you guys made an "open letter", seems like admins are trying very hard to stop, close and delete every argument or consideration they receive as github issue about all of this.

So... what's the reason of making open letters if you aren't open to discuss anything? Many people or institutions who signed the letter even set their social networks as private to avoid talking about it.

How people who made and sign this letter may claim to belong into a community, if they aren't even making space to discuss? What kind of community do you belong? Is this just the mob rule?

This sadly looks like purely witch hunting, no discussions, no reasoning, no arguments, just hunt and kill the witch because we don't like em. Playing Robespierre game it's a way dangerous path.

**Comment by waltercool** [comment]
Interestingly, this is issue number #2259, but there are only 36 issues in existence. 3 open, 33 closed.

What happened with all 2222 others? Any specific reason to purge closed issues and leave no traces about it?

**Comment by Watcher3056** [comment]
> 
> 
> Interestingly, this is issue number #2259, but there are only 36 issues in existence. 3 open, 33 closed.
> 
> What happened with all 2222 others? Any specific reason to purge closed issues and leave no traces about it?

You can see the same situation with issue index in rms-support-letter. Seems like its not linked directly to its count.

**Comment by Watcher3056** [comment]
But i agree with your upper decision

**Comment by Watcher3056** [comment]
I think issue index linked to last pull request index, when it was created

**Comment by waltercool** [comment]
Oh yes, you are right about it. So, 2222 - 2165 (sum all pull requests) = means there are at least 57 "issues/pull" which are missing.

Just check this out at the other letter, and quite same, 2377 (closed pull requests) + 81 (open PR) +25 (open issues) + 44 (closed issues). Last id = 2543. That leaves 16 missing issues/pull.

I mean, if there is an open letter, I expect transparency on all doings, this also includes they should be open for discussion and listen both sides. Making two different "open letters" who don't talk each other, doesn't seems very democratic.

**Comment by 6r1d** [comment]
> Making two different "open letters" who don't talk each other, doesn't seems very democratic.

This situation looks weird from the "support letter" perspective.
"[continuation: rms-open-letter denial of discussion](https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175)"

**Comment by waltercool** [comment]
> > Making two different "open letters" who don't talk each other, doesn't seems very democratic.
> 
> This situation looks weird from the "support letter" perspective.
> "[continuation: rms-open-letter denial of discussion](https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175)"

Yep, but at least they have "Discussions" open. I opened a discussion at the other letter about the same topic. Sorry to open an "issue" here, but there is no other option.

Maintaining two positions without discussion won't solve anything.

**Comment by CarlosGS** [comment]
They are actively removing any discussions here, see https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175 for a log on the closed. I don't mind them closing issues, but fully deleting relevant ones is going too far.

**Comment by waltercool** [comment]
> They are actively removing any discussions here, see [rms-support-letter/rms-support-letter.github.io#175](https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175) for a log on the closed ones. I don't mind them closing issues, but fully deleting relevant ones is going too far.

100% on it.

If you are making letter for free software issues, at least transparency and accountability is needed. That's the reason why most FSF projects (with mailing lists) keeps every communication open and public.
